src/mumble/PulseAudio.cpp

```diff
@@ -566,15 +566,23 @@ void PulseAudioSystem::write_callback(pa_stream *s, size_t bytes, void *userdata
 	PulseAudioSystem *pas = reinterpret_cast< PulseAudioSystem * >(userdata);
 	Q_ASSERT(s == pas->pasOutput);
 
+	const auto &pa = pas->m_pulseAudio;
```

Comment:
Use of `auto` violates our coding guidelines - see https://github.com/mumble-voip/mumble/blob/master/CODING_GUIDELINES.md#use-of-auto

```suggestion
	const PulseAudio &pa = pas->m_pulseAudio;
```